.env.example

```
@@ -1,3 +1,4 @@
-VITE_APPWRITE_ENDPOINT=
-VITE_APPWRITE_PROJECT_ID=
-VITE_APPWRITE_PROJECT_NAME=
\ No newline at end of file
+VITE_APPWRITE_PROJECT_ID = "699ebd17003273658385"
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)